packages/language-server/src/diagnosticsHandler.ts

```diff
@@ -0,0 +1,34 @@
+import { Diagnostic, DiagnosticSeverity, DiagnosticTag } from 'vscode-languageserver/node'
+import { getBlockAtPosition } from './util'
+
+export function greyOutIgnoredParts(lines: string[]): Diagnostic[] {
+  const diagnostics: Diagnostic[] = []
+
+  lines.map((currElement, index) => {
+    if (currElement.includes('@@ignore')) {
+      const block = getBlockAtPosition(index, lines)
+      if (block) {
+        diagnostics.push({
+          range: { start: block.range.start, end: block.range.end },
+          message:
+            '@@ignore: this model will be kept in sync with the database schema, however, it won’t be exposed to the Prisma client.',
```

Comment:
```suggestion
            '`@@ignore`: When using Prisma Migrate, this model will be kept in sync with the database schema, however, it will not be exposed in Prisma Client.',
```

Reply:
This seems to be possible @janpio, currently reading through [this issue thread](https://github.com/Microsoft/vscode/issues/11847) to learn more

Reply:
Seems like there was a proposed API, and a PR that had some implementation but it was closed, _not_ merged. So this is still something that vscode doesn't support.

[proposed api](https://github.com/microsoft/vscode/issues/11847#issuecomment-578055139)
[closed pr](https://github.com/microsoft/vscode/pull/61436)

Reply:
It looks like it should be available though https://code.visualstudio.com/updates/v1_42#_adding-links-to-diagnostic-errors
Code from main branch https://github.com/microsoft/vscode/blob/1a69d6cb15f3885eb1c0cb6067f8b907f8268775/src/vscode-dts/vscode.d.ts#LL6157C4-L6157C11

Reply:
Hm, strange, this is what I have available on inspect of the type `Diagnostic`
<img width="539" alt="image" src="https://user-images.githubusercontent.com/29753584/201902259-e53c49ed-38f7-462b-873b-4a13dab5c7dd.png">

Reply:
TLDR from Slack, we could try this:
https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#diagnostic
```
	/**
	 * An optional property to describe the error code.
	 *
	 * @since 3.16.0
	 */
	codeDescription?: CodeDescription;
```
https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#codeDescription
```
/**
 * Structure to capture a description for an error code.
 *
 * @since 3.16.0
 */
export interface CodeDescription {
	/**
	 * An URI to open with more information about the diagnostic error.
	 */
	href: URI;
}
```